webflow-hacks/misc-hacks/22/22.ts

```diff
@@ -0,0 +1,27 @@
+// on DOM ready
+document.addEventListener('DOMContentLoaded', function () {
+  // selectors
+  const DATE_VERSION_ONE_SELECTOR = '[fs-hacks-element="date-version1"]';
+  const DATE_VERSION_TWO_SELECTOR = '[fs-hacks-element="date-version2"]';
+  const dateVersionOne = document.querySelector<HTMLHeadingElement>(DATE_VERSION_ONE_SELECTOR);
+  const dateVersionTwo = document.querySelector<HTMLHeadingElement>(DATE_VERSION_TWO_SELECTOR);
+  // early exit if no elements found
+  if (!dateVersionOne || !dateVersionTwo) return;
+  //set your formatting options
+  //Version 1 output = "Sat, Dec 25, 2019"
```

Comment:
Can we add a link reference to the MDN docs for `Intl.DateTimeFormatOptions`?
This way users can explore more options.

Something like:
```typescript
// See more available options at {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#options}
```

Reply:
done.